### ISSUE TITLE
[QUESTION] How to install this thing if setup not working?

### ISSUE THREAD

**Issue by Gh0sTG0** [Alexhuszagh/BreezeStyleSheets#87]
Hi. Using pyqt6 on python 3.11. Coding with pycharm.
I downloaded full code as a zip, then placed folder from the archive inside my project folder, full path for folder is like `D:\My\Project\BreezeStyleSheets` and all that `assets`, `dist`, etc. folders inside.
Then I'm going to this part: https://github.com/Alexhuszagh/BreezeStyleSheets#pyqt6-installation
I created an empty `test.py` file inside the project (it is on one level with `BreezeStyleSheets` folder) and placed there:
```
from setuptools import setup

setup(
    # Either option is valid here.
    #   Either use `package_data` with enumerating the values, or
    #   set `include_package_data=True`.
    # include_package_data=True,
    package_data={
        "breeze_theme": ["BreezeStyleSheets/dist/pyqt6/*"],
    },
    zip_safe=False,
)

```
And it crashes with:
```
usage: test.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: test.py --help [cmd1 cmd2 ...]
   or: test.py --help-commands
   or: test.py cmd --help

error: no commands supplied
```
Where I'm incorrect in what I'm doing?

PS I tried commenting `include_package_data` or `package_data` rows, nothing changed.

**Comment by Alexhuszagh** [comment]
The documentation probably needs to be updated, will do tonight.

**Comment by Alexhuszagh** [comment]
This is resolved with the PR #89 and the documentation enhancements in #92.
